counterpoll/main.py

```diff
@@ -241,6 +241,40 @@ def disable():
     configdb.mod_entry("FLEX_COUNTER_TABLE", "PG_WATERMARK", fc_info)
     configdb.mod_entry("FLEX_COUNTER_TABLE", BUFFER_POOL_WATERMARK, fc_info)
 
+# Tunnel counter commands
+@cli.group()
+def tunnel():
+    """ Tunnel counter commands """
+
+@tunnel.command()
+@click.argument('poll_interval')
+def interval(poll_interval):
+    """ Set tunnel counter query interval """
+    configdb = ConfigDBConnector()
+    configdb.connect()
+    tunnel_info = {}
+    if poll_interval is not None:
+        tunnel_info['POLL_INTERVAL'] = poll_interval
+    configdb.mod_entry("FLEX_COUNTER_TABLE", "TUNNEL", tunnel_info)
+
+@tunnel.command()
+def enable():
+    """ Enable tunnel counter query """
+    configdb = ConfigDBConnector()
+    configdb.connect()
+    tunnel_info = {}
+    tunnel_info['FLEX_COUNTER_STATUS'] = 'enable'
```

Comment:
There is already defined ```ENABLE``` global variable in this file for the "enable" string, suggest to use it here.